src/index.html

```diff
@@ -1,13 +1,508 @@
+                    </li>
+                    <li class="nav__item">
+                      <a href="#events" class="nav__link">Виставки та події</a>
+                    </li>
+                  </div>
+                  <div class="nav__container">
+                    <li class="nav__item">
+                      <a href="#describe" class="nav__link">Про Нас</a>
+                    </li>
+                    <li class="nav__item">
+                      <a href="#contacts" class="nav__link">Контакти</a>
+                    </li>
+                  </div>
+                </ul>
+              </div>
+              <div class="schedule menu__schedule">
+                <div class="schedule__container">
+                  <p class="schedule__title">Графік роботи</p>
+                  <ul class="schedule__list">
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">12:00 — 20:00</span> середа
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">12:00 — 20:00</span> четвер
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">12:00 — 20:00</span> п’ятниця
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">11:00 — 19:00</span> субота
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">11:00 — 19:00</span> неділя
+                    </li>
+                  </ul>
+                </div>
+                <p class="schedule__weekend">
+                  <span class="schedule__weekend--strong">Вихідні:</span> понеділок, вівторок<br> Каса припиняє роботу за 1 годину до закриття музею
+                </p>
+              </div>
+            </div>
+            <div class="logo top-bar__logo">
+              <a href="#home">na<br>mu</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="header__overlay"></div>
+    </header>
+    <main class="main">
+      <section class="section">
+        <div class="section__container">
+          <div class="hero js-anim-items">
+            <div class="hero__overlay">
+              <img
+                src="./images/hero/hero-mobile.jpg"
+                srcset="./images/hero/hero-mobile.jpg 320w, ./images/hero/hero-1x.jpg 400w, ./images/hero/hero-2x.jpg 750w"
+                alt="Фото музею"
+                class="hero__image"
+                sizes="(max-width: 767px) 320px, (max-width: 1279px) 400px, (min-width: 1280px) 750px"
+              >
+            </div>
+            <div class="hero__info">
+              <h1 class="hero__title anim-show js-anim-no-hide">Художній музей</h1>
+              <a href="#describe" class="button hero__button anim-show js-anim-no-hide">
```

Comment:
Consider improving naming of class because you dont use button in this case, you use link
Change it to link-button or something similar or wrap link in a button tag

Check in the other similar case

Reply:
Yes in this case I use tag a, because this button go to other block. I think about wrap link in button or button in link, but in this case, html become too unlogical, so I desided leave this variant with a. It is only in hero section. and all styles the same as button, I think it is better then wrap one button in link.